tRecorderApi/api/models/take.py

```diff
@@ -14,20 +15,20 @@ class Take(models.Model):
     class Meta:
         ordering = ["chunk"]
 
-    def __str__(self):
-        return '{} ({})'.format(self.chunk, self.id)
+    # def __str__(self):
+    #     return '{} ({})'.format(self.chunk, self.id)
 
-    # @staticmethod
-    # def stitchSource(data):
-    #     list = []
-    #     filter = {}
-    #     filter["chapter__project__language__slug"] = data["language"]
-    #     filter["chapter__project__version"] = data["version"]
-    #     filter["chapter__project__book__slug"] = data["book"]
-    #     filter["chapter__number"] = data["chapter"]
+    @staticmethod
+    def stitchSource(data):

```

Comment:
if this method is being kept, the name should reflect what it's doing (getting a list of chunks)

Reply:
I  did that when I was playing around with the code and forgot to comment it back. I did not really work on that file